src/compiler/diagnosticMessages.json

```diff
@@ -4945,7 +4944,7 @@
         "category": "Message",
         "code": 6259
     },
-    "Found {0} errors in 1 file.": {
+    "Found {0} errors in 1 file: {1}": {

```

Comment:
Not opposed to this, but also consider just:

```suggestion
    "Found {0} errors in {1}": {
```

Reply:
I agree, that is tighter, will push a commit